tax-tracker/src/data/studentLoan.js

```diff
@@ -9,5 +9,5 @@ export const studentLoan = ({ salary, type, year }) => {
 
   if (salary < studentLoanThreshold) return 0;
   
-  return (salary - studentLoanThreshold) * 0.09
+  return (salary - studentLoanThreshold) * slRate[type + "_rate"];
```

Comment:
Instead of this you could organise the object like 
```
[year]: {
   plan_1: {
       rate: 0.09
   },...
}
```
then you could access by `slRate[type].rate;`

Reply:
Sounds gooooood re, lets do it that wayyyy! Some noSql right there :P